src/Tizen.NUI.Components/Controls/ItemsView/ItemsView.cs

```diff
@@ -345,110 +283,65 @@ protected override float AdjustTargetPositionOfScrollAnimation(float position)
-
-        }
-
-        /// <summary>
-        /// </summary>
-        /// <param name="source"></param>
-        /// <param name="startIndex"></param>
-        /// <since_tizen> 8 </since_tizen>
-        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
-		public void NotifyItemInserted(IItemSource source, int startIndex)
-        {
-
-        }
-
         /// <summary>
+        /// Push the item into the recycle cache. this item will be reused in view update.
         /// </summary>
-        /// <param name="source"></param>
-        /// <param name="fromPosition"></param>
-        /// <param name="toPosition"></param>
-        /// <since_tizen> 8 </since_tizen>
-        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
-		public void NotifyItemMoved(IItemSource source, int fromPosition, int toPosition)
+        /// <param name="item"> Target item to push into recycle cache. </param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected virtual bool PushRecycleCache(ViewItem item)
         {
-
+            if (item == null) throw new ArgumentNullException(nameof(item));
+            if (RecycleCache.Count >= CacheMax) return false;
+            if (item.Template == null) return false;
+            item.Hide();
+            item.Index = -1;
+            RecycleCache.Add(item);
+            return true;
         }
 
         /// <summary>
+        /// Pop the item from the recycle cache.
         /// </summary>
-        /// <param name="source"></param>
-        /// <param name="start"></param>
-        /// <param name="end"></param>
-        /// <since_tizen> 8 </since_tizen>
-        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
-		public void NotifyItemRangeChanged(IItemSource source, int start, int end)
+        /// <param name="Template"> Template of wanted item. </param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected virtual ViewItem PopRecycleCache(DataTemplate Template)
         {
-
+           for (int i = 0; i < RecycleCache.Count; i++)
+           {
+               ViewItem item = RecycleCache[i];
+               if (item.Template == Template)
+               {
+                   RecycleCache.Remove(item);
+                   item.Show();
+                   return item;
+               }
+           }
+           return null;
         }
 
-        /// <summary>
-        /// </summary>
-        /// <param name="source"></param>
-        /// <param name="startIndex"></param>
-        /// <param name="count"></param>
-        /// <since_tizen> 8 </since_tizen>
-        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
-		public void NotifyItemRangeInserted(IItemSource source, int startIndex, int count)
+        private void OnScrolling(object source, ScrollEventArgs args)
```

Comment:
How about changing this to protected virtual to let CollectionView override this function?